docs/changelog.rst

```diff
@@ -25,6 +25,7 @@ Develop
 * [DEPRECATION] Deprecate demo suite behavior in `suite new`
 * [FEATURE] Slack Messages sent as ValidationActions now have link to DataDocs, if available.
 * [FEATURE] ValidationActions can now consume and return "payload", which can be used to share information across ValidationActions
+* [ENHANCEMENT] Add support for combining schema with anyOf key in JSON Schema profiler
```

Comment:
I would also add that the profiler makes nullity expectations as well - a key new feature!

Reply:
Good point, now done!